pkg/controller/controller.go

```diff
@@ -208,6 +216,7 @@ func (c *controller) reconcileBroker(broker *v1alpha1.Broker) {
 		}
 
 		c.updateBrokerReadyCondition(broker, v1alpha1.ConditionTrue, "FetchedCatalog", "Successfully fetched catalog from broker.")
+		c.recorder.Eventf(broker, api.EventTypeNormal, "BrokerReady", "Successfully fetched catalog entries from broker")
```

Comment:
I would use `FetchedCatalog` as the reason here, and extract the reason into a string constant.  Extract all the string literal reasons into constants and @arschles will love you.  Likewise with any event messages that are just repeated literals (like `Successfully fetched catalog entries for broker` here).